charts/kpow/templates/deployment.yaml

```diff
@@ -37,7 +37,7 @@ spec:
           env:
             {{- range $key, $val := . }}
             - name: {{ $key }}
-              value: {{ $val }}
```

Comment:
Discovered this when I was trying to set `SAMPLER_CONSUMER_THREADS`
Throws an error when setting any environment variable to an integer value.  This fixed it.